crates/nu-lsp/src/lib.rs

```diff
@@ -1017,10 +1023,11 @@ mod tests {
             .as_str()
             .unwrap()
             .to_string();
+
         #[cfg(not(windows))]
         assert!(hover_text.starts_with("SLEEP(1)"));
         #[cfg(windows)]
-        assert!(hover_text.starts_with("No manpage found for sleep"));
+        assert!(hover_text.starts_with("NAME\r\n    Start-Sleep"));
```

Comment:
I was debating on whether to leave this as `NAME` only or do `hover_text.contains("Start-Sleep")`. I'll let you be the judge on what is best.

Reply:
This looks good to me